api/ccapi/include/layer.h

```diff
@@ -111,19 +111,23 @@ enum LayerType {
     ML_TRAIN_LAYER_TYPE_LOSS_CROSS_ENTROPY_SOFTMAX, /**< Cross Entropy with
                                                        Softmax Loss Layer type
                                                      */
-  LAYER_TIME_DIST,       /**< Time Distributed Layer type */
-  LAYER_BACKBONE_TFLITE, /**< Backbone using TFLite */
-  LAYER_RESHAPE,         /**< Reshape Layer type */
-  LAYER_REDUCE_MEAN,     /**< Reduce mean Layer type */
+  LAYER_RMSNORM = ML_TRAIN_LAYER_TYPE_RMSNORM,      /**<RMS NORM Layer */
+  LAYER_TRANSPOSE = ML_TRAIN_LAYER_TYPE_TRANSPOSE,  /**< Transpose Layer type */
+  LAYER_CHANNEL_SHUFFLE =
+    ML_TRAIN_LAYER_TYPE_CHANNEL_SHUFFLE, /**< Channel Shuffle Layer type */
   LAYER_REDUCE_SUM =
     ML_TRAIN_LAYER_TYPE_REDUCE_SUM, /**< Reduce sum Layer type */
-  LAYER_LOSS_CONSTANT_DERIVATIVE,   /**< Synthetic loss layer to feed constant
-                                       derivative */
-  LAYER_UPSAMPLE2D,                 /**< Upsample 2D Layer type */
-  LAYER_RMSNORM = ML_TRAIN_LAYER_TYPE_RMSNORM,     /**<RMS NORM Layer */
-  LAYER_TRANSPOSE = ML_TRAIN_LAYER_TYPE_TRANSPOSE, /**< Transpose Layer type */
-  LAYER_CHANNEL_SHUFFLE =
-    ML_TRAIN_LAYER_TYPE_CHANNEL_SHUFFLE,      /**< Channel Shuffle Layer type */
+  LAYER_REDUCE_MEAN =
+    ML_TRAIN_LAYER_TYPE_REDUCE_MEAN,           /**< Reduce mean Layer type */
+  LAYER_RESHAPE = ML_TRAIN_LAYER_TYPE_RESHAPE, /**< Reshape Layer type */
+  LAYER_UPSAMPLE2D =
+    ML_TRAIN_LAYER_TYPE_UPSAMPLE2D, /**< Upsample 2D Layer type */
+
+  LAYER_TIME_DIST = 1000,                /**< Time Distributed Layer type */
```

Comment:
If '1000' doesn't mean much, wouldn't it be okay to start the value at 900 since the 'LAYER_UNKNOWN' value is 999?